Goal: Amend the LDAP/README example as smbldap-tools are packaged separately

Fixes: #341934

Status wrt upstream: Debian specific

Note: 

Index: samba-3.0.25c/examples/LDAP/README
===================================================================
--- samba-3.0.25c.orig/examples/LDAP/README	2007-08-26 12:07:16.598298995 +0200
+++ samba-3.0.25c/examples/LDAP/README	2007-08-26 13:09:02.419701485 +0200
@@ -59,6 +59,9 @@
 The smbldap-tools package can be downloaded individually from 
 http://samba.idealx.org/dist/
 
+On Debian systems, the smbldap-tools exists as a separate package
+and is not included in LDAP examples.
+
 !==
 !== end of README
 !==
